cppcache/integration-test/testThinClientSecurityAuthorizationMU.cpp

```diff
@@ -160,6 +160,7 @@ void initClientAuth(char UserType) {
     case 'A':
       credentialGeneratorHandler->getAllowedCredentialsForOps(ad, userCreds,
                                                               nullptr);
+      break;
     default:
```

Comment:
What's with the use of `default` here but in other places it is replaced with explicit `case` statements?

Reply:
`default` is used here because this switch statement is checking string characters.  The others are checking an enum.  Since an enum has a well defined set of values, you do not need a default if you have a case statement for each value.  `no-covered-switch-default` and `no-switch-enum` help enforce that